build_projects/Microsoft.DotNet.Cli.Build.Framework/BuildFailureException-BuildTarget.cs

```diff
@@ -0,0 +1,29 @@
+using System;
```

Comment:
Do you have a suggestion? It's a partial class which encapsulates the piece of the class that has a dependency on BuildTarget.

This way I don't need to source include the entire build system in the tests.